scripts/firestore_delete/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Function dependencies, for example:
+# package>=version
+google-cloud-firestore
```

Comment:
Is the package not available by default in the Cloud Functions environment?

Reply:
No it is needed to be mentioned in the requirements.txt file, else we get an error